docs/development/evaluations/latest-results.md

```
@@ -4,7 +4,7 @@ Redirecting to the latest benchmark results...
 
 <script>
 
-window.location.href = "../history/results_20260315_041151/";
+window.location.href = "history/results_20260315_041151/";
 </script>
 
-If you are not redirected automatically, [click here](../history/results_20260315_041151/).
+If you are not redirected automatically, [click here](history/results_20260315_041151.md).
```

markdownlint-cli2 (0.22.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)